src/Manager/FigureManager.php

```diff
@@ -20,6 +20,16 @@ public function __construct(private SnowFigureRepository $snowFigureRepository,
         $this->slugger = $slugger;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getListFigures(): array
+    {
+        $figuresList = $this->snowFigureRepository->findBy([], ['id' => 'DESC']);
```

Comment:
en une seule ligne ? 
```suggestion
        return $this->snowFigureRepository->findBy([], ['id' => 'DESC']);
```

Reply:
done